packages/gafl-webapp-service/src/handlers/analytics-handler.js

```diff
@@ -6,17 +6,27 @@ import { ANALYTICS } from '../constants.js'
  * @returns {Promise}
  */
 
-export const checkAnalytics = async request => {
+export const checkAnalytics = async (request, pageSkip) => {
   try {
     const analytics = await request.cache().helpers.analytics.get()
-    if (analytics && analytics[ANALYTICS.acceptTracking] === true) {
+    if (analytics[ANALYTICS.acceptTracking] === true && pageSkip !== true) {
       return true
     }
   } catch {}
 
   return false
 }
 
+export const checkPage = async request => {
```

Comment:
This needs a better name than 'checkPage' - I'd say the same is true for 'checkAnalytics' too. Look at exactly what each function is checking and base the name on that